CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing Guidelines
+
+## Create a grammar
+
+We recommended creating the grammar using our [template], either via the
+"Use this template" button on the web interface, or with the `--template` flag
+of the [gh cli]. Wait a moment for the initialization to complete before cloning
+the repository. The default license is MIT but you can change it to whatever you want.
+
+Make sure to read the Tree-sitter [documentation] before you start
+developing the grammar. The template handles most of the project setup
+for you. Just run these three commands in the grammar directory to get started:
```

Comment:
```suggestion
for you. Just run these two commands in the grammar directory to get started:
```

Reply:
I can't even remember what the third command was before I removed it…